### ISSUE TITLE
When to choose between Duration and TimeSpan?

### ISSUE THREAD

**Issue by R2D221** [nodatime/nodatime#1585]
Most of the time, it's obvious why NodaTime is superior to the standard library, with types like LocalTime or ZonedDateTime or such that have no direct standard library counterpart.

But, with Duration, I've read the documentation and the user guide, and I still see how exactly this is different from TimeSpan. Yes, apparently Duration has more precision, but other than that, both represent the same concept, or am I missing something?

So, if I want to represent the concept of “5 seconds”, which type should I choose?

**Comment by jskeet** [comment]
`TimeSpan` and `Duration` are conceptually identical, yes (beyond the precision). Before 1.0 we even had a discussion about whether to include `Duration` at all or just use `TimeSpan` - we decided to include `Duration` for more flexibility, and I'm glad we did. (Otherwise we wouldn't have been able to move to nanosecond precision, for example.)

As for choosing between the two: if you're working with NodaTime types, use `Duration`. If you're working with BCL types, use `TimeSpan`.
